exporter/exporterparser/zipkin_test.go

```diff
@@ -0,0 +1,228 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package exporterparser_test
+
+import (
+	"bytes"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/census-instrumentation/opencensus-service/exporter"
+	"github.com/census-instrumentation/opencensus-service/exporter/exporterparser"
+	"github.com/census-instrumentation/opencensus-service/interceptor/zipkin"
+	"github.com/census-instrumentation/opencensus-service/internal"
+)
+
+// This function tests that Zipkin spans that are intercepted then processed roundtrip
+// back to almost the same JSON with differences:
+// a) Go's net.IP.String intentional shortens 0s with "::" but also converts to hex values
+//    so
+//          "7::0.128.128.127"
+//    becomes
+//          "7::80:807f"
+// b) OpenZikpin-Go unconditionally creates a non-blank IPv4 if only IPv6 addresses were resolved
```

Comment:
this should be fixed per: https://github.com/openzipkin/zipkin-go/pull/86

Reply:
Perfect, thank you @basvanbeek for the quick response and fix!